Makefile

```diff
@@ -52,11 +52,11 @@ all:
 	@echo "Current type detected is: $(type)"
 
 recap.cron:
-	@sed -e 's|@BINDIR@|$(BINDIR)|' src/utils/recap.cron.in > src/utils/recap.cron
+	@sed -e 's|\(^BINDIR=\).*|\1$(BINDIR)|' src/utils/recap.cron.in > src/utils/recap.cron
```

Comment:
This one made me think a bit, usually I use the `^` outside the matching group., let's also include the terminator `$`.

Reply:
Fixed, thanks!